Lib/idlelib/idle_test/test_browser.py

```diff
@@ -0,0 +1,236 @@
+""" Test idlelib.browser.
+"""
```

Comment:
I should add coverage.  100% requires a mock that raises specific exception.  I know mock_idle.Func can and presume the unittest mock can also.